### PR TITLE
LDAP Profile Picture Sync attribute

### PR DESCRIPTION
**Pull request by thefactremains**
details on AD/LDAP profile picture sync

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2212,6 +2212,16 @@ Email notifications will be sent to this email address, and this email address m
 | This feature's ``config.json`` setting is ``"EmailAttribute": ""`` with string input.                                                                                |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Profile Picture Attribute
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The attribute in the AD/LDAP server used to synchronize (and lock) the profile picture used in Mattermost.
+
+The mattermost server will replace the user’s profile image upon login. Note, this only occurs when the user either does not have a profile image in mattermost, or the current profile image does not match the image associated with that user in AD/LDAP.
```
Comment (sbishel) [inline source/administration/config-settings.rst:2219]: Should mattermost be capitalized? 2 places. The statement after "Note:" seems confusing. "This ONLY occurs...", when else would it occur?
Should there be something about occurs at login, not with normal sync.

Comment (justinegeffen) [inline source/administration/config-settings.rst:2219]:
```suggestion
The Mattermost server will replace the user’s profile image upon login. This only occurs when the user either does not have a profile image in Mattermost, or the current profile image does not match the image associated with that user in AD/LDAP.
```

Comment (thefactremains) [inline source/administration/config-settings.rst:2219]:
```suggestion
The Mattermost server will replace the user’s profile image upon login (not at the sync interval as with other attributes). The sync will not occur if the current profile image matches the image associated with that user in AD/LDAP.
```

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2212,6 +2212,16 @@ Email notifications will be sent to this email address, and this email address m
 | This feature's ``config.json`` setting is ``"EmailAttribute": ""`` with string input.                                                                                |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Profile Picture Attribute
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The attribute in the AD/LDAP server used to synchronize (and lock) the profile picture used in Mattermost.
+
+The Mattermost server will replace the user’s profile image upon login (not at the sync interval as with other attributes). The sync will not occur if the current profile image matches the image associated with that user in AD/LDAP.
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:2219]:
```suggestion
The Mattermost server will replace the user’s profile image upon login (not at the sync interval as with other attributes). The sync will not occur if the current Mattermost profile image matches the image associated with that user in AD/LDAP.
```

**Pull request merged**